addOns/ascanrules/src/main/java/org/zaproxy/zap/extension/ascanrules/PathTraversalScanRule.java

```diff
@@ -75,8 +98,6 @@ public class PathTraversalScanRule extends AbstractAppParamPlugin {
         "..\\..\\..\\..\\..\\..\\..\\..\\..\\..\\..\\..\\..\\..\\..\\..\\Windows\\system.ini",
         "/../../../../../../../../../../../../../../../../Windows/system.ini",
         "\\..\\..\\..\\..\\..\\..\\..\\..\\..\\..\\..\\..\\..\\..\\..\\..\\Windows\\system.ini",
-        // "../../../../../../../../../../../../../../../../Windows/system.ini%00.html",
-        // "..\\..\\..\\..\\..\\..\\..\\..\\..\\..\\..\\..\\..\\..\\..\\..\\Windows\\system.ini%00.html",
```

Comment:
I just removed the commented payloads using a null byte injection, because they are already executed in the current code.
All other commented payloads contain further/new payloads which could be added in an upcoming enhancement

Reply:
I think you can trim the others too.